protos/common/window.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package common;
+
+import "google/protobuf/timestamp.proto";
+
+// Sets the golang package for the protobuf generated code
+option go_package = "github.com/opencost/opencost/core/pkg/common/pb";
```

Comment:
I am not sure about the disparate packaging on this protos. maybe they should all be something like "github.com/opencost/opencost/core/pkg/model/pb/common|kubemodel|customcost" @ameijer @nikovacevic thoughts?

Reply:
I agree with you sean, I like the proposed file structure